backend/src/main/scala/cromwell/backend/WorkflowBackendActor.scala

```diff
@@ -0,0 +1,77 @@
+package cromwell.backend
+
+import akka.actor.{ActorLogging, Actor}
+import akka.event.LoggingReceive
+import cromwell.backend.WorkflowBackendActor._
+import cromwell.core.CallOutputs
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success}
+import WorkflowBackendActor.ExecutionFailed
+
+object WorkflowBackendActor {
+
+  // Commands
+  sealed trait WorkflowBackendActorMessage
+
+  case class Execute(jobDescriptor: BackendJobDescriptor) extends WorkflowBackendActorMessage
+  case class Recover(jobDescriptor: BackendJobDescriptor) extends WorkflowBackendActorMessage
+  case class Abort(jobKey: BackendJobDescriptorKey) extends WorkflowBackendActorMessage
+
+  // Responses
+  sealed trait WorkflowBackendActorResponse
+
+  sealed trait ExecutionResponse extends WorkflowBackendActorResponse
+  case class ExecutionSucceeded(jobKey: BackendJobDescriptorKey, callOutputs: CallOutputs) extends ExecutionResponse
+  case class ExecutionFailed(jobKey: BackendJobDescriptorKey, throwable: Throwable) extends ExecutionResponse
```

Comment:
Since both exceptions contains the same, can we have something like this?
` ExecutionFailed(jobKey: BackendJobDescriptorKey, throwable: Throwable, retryable: Boolean = false) `